localization/yabloc/yabloc_common/include/yabloc_common/color.hpp

```diff
@@ -31,25 +31,28 @@ struct Color
   }
 
   explicit Color(const cv::Scalar & rgb, float a = 1.0f)
-  : r(rgb[2] / 255.f), g(rgb[1] / 255.f), b(rgb[0] / 255.f), a(a)
+  : r(static_cast<float>(rgb[2]) / 255.f),
+    g(static_cast<float>(rgb[1]) / 255.f),
+    b(static_cast<float>(rgb[0]) / 255.f),
+    a(a)
   {
   }
 
-  operator uint32_t() const
+  explicit operator uint32_t() const
```

Comment:
Since implicit type conversion to uint32_t is prohibited, yabloc_particle_filter package is causing compilation errors. Please add `static_cast<uint32_t>` to the following 2 lines:


https://github.com/autowarefoundation/autoware.universe/blob/f25396f8bfabee250d03d4affec912623c7d9f95/localization/yabloc/yabloc_particle_filter/src/camera_corrector/camera_particle_corrector_core.cpp#L209

https://github.com/autowarefoundation/autoware.universe/blob/f25396f8bfabee250d03d4affec912623c7d9f95/localization/yabloc/yabloc_particle_filter/src/camera_corrector/camera_particle_corrector_core.cpp#L215


FYI, you can build all yabloc packages by
`colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release -DCMAKE_EXPORT_COMPILE_COMMANDS=ON --packages-select-regex yabloc`

Reply:
Thank you for pointing out the issue. It seems I forgot to check the full compilation.
It is fixed in ffd037c .